### PR TITLE
Get Posts By Controller / Community Entity

### PR DESCRIPTION
**Pull request by kiymetakdemir**
Changed method of getting posts. Instead of getting by id, gets by name. 
Deleted duplicated dependency.
Deleted getters in community entity.

### CODE REVIEW

**Review by irem-zeynep — APPROVED** [review]
Looks good, I just suggested a different name for a variable. Thanks!

File: practiceapp/src/main/java/com/practiceapp/practiceapp/controller/community/GetPostsController.java [hunk 1]
```diff
@@ -21,8 +21,8 @@
     @Autowired
     private CommunityService communityService;
 
-    @RequestMapping(path = "/posts/{communityid}",method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
-    public List<PostEntity> getPosts(@PathVariable("communityid") String communityid){
+    @RequestMapping(path = "/posts/{name}",method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public List<PostEntity> getPosts(@PathVariable("name") String communityid){
```
[the remark this answers is not in the document]
Comment (kiymetakdemir) [reply practiceapp/src/main/java/com/practiceapp/practiceapp/controller/community/GetPostsController.java]: I changed it. Thank you!

**Review by irem-zeynep — APPROVED** [review]
It makes sense to use the community name instead of the id. Thanks.

**Comment by orhungorkem** [comment]
Name is more user friendly. I see you also updated service as well as controller. There seems to be no problem. Thank you.

**Pull request merged**